tests/test_sparse.py

```
@@ -0,0 +1,61 @@
+        # Verify select was called with appropriate parameters
+        mock_db_session.exec.assert_called_once()
+
+    def test_retrieve_empty_query(self, sparse_retrieval, mock_db_session):
+        # Test with an empty query
+        result = sparse_retrieval.retrieve("")
+        
+        # Assert the result is an empty list
+        assert result == []
+        
+        # Verify select was not called
+        mock_db_session.exec.assert_not_called
```

Ruff (0.8.2)

Found useless expression. Either assign it to a variable or remove it.

(B018)